README.rst

```diff
@@ -3,7 +3,7 @@
 plothist
 ========
 
-**Plot histograms in a scalable way and a beautiful style.**
+**Visualize data in a scalable way and a beautiful style.**
```

Comment:
It can do more than just visualize, but I think this verb is broad enough.

Reply:
Yes, I wanted to stay short. What about Visualize and compare...?

Reply:
Agreed.